src/pages/transactions/TransactionForm.js

```diff
@@ -133,7 +138,9 @@ export default function TransactionForm({ onAddTransaction, onAlert }) {
           </select>
         </div>
         {/* Submit button */}
-        <button type="submit">Add</button>
+        <Button variant="primary" endIcon="add" onClick={onAddTransaction}>
+          Add
+        </Button>
```

Comment:
QA issues: when being redirected to the homepage after adding an entry, it looks like this: 
![image](https://github.com/miToe/dagobert/assets/123096081/48b7d3a6-5fd7-42bb-ace5-9f7708773bc4)